terraform-provider-constellation/internal/provider/cluster_resource_test.go

```diff
@@ -23,9 +24,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var providerVersion = semver.NewFromInt(2, 15, 0, "")
+var providerVersion string
```

Comment:
```suggestion
```
If we dont assign a value, we should not have it package scoped to prevent accidentally setting a value and in that way breaking other tests

Reply:
this value needs to be injected in the build (see bazel) to make the microservice version compatible with the provider build

Reply:
But this is in the unit test.
If the value being injected by bazel is required, then this breaks running Go the tests outside of bazel.

And as far as I can tell, we simply override the value in some functions, e.g. [here](https://github.com/edgelesssys/constellation/blob/7ce339d343e28e58b0ce64694a3ed23c38b60cdd/terraform-provider-constellation/internal/provider/cluster_resource_test.go#L30).
Whats stopping us from doing the same in other tests as well?